source/get-started/pojo-qs.txt

```diff
@@ -0,0 +1,410 @@
+       private int age;
+       private Address address;
+   
+       public Person() {
+       }
+   
+       public ObjectId getId() {
+           return id;
+       }
+   
+       public void setId(final ObjectId id) {
+           this.id = id;
+       }
+   
+       public String getName() {
+           return name;
+       }
+   
+       public void setName(final String name) {
+           this.name = name;
+       }
+   
+       public int getAge() {
+           return age;
+       }
+   
+       public void setAge(final int age) {
+           this.age = age;
+       }
+   
+       public Address getAddress() {
+           return address;
+       }
+   
+       public void setAddress(final Address address) {
+           this.address = address;
+       }
+       
+       // Rest of implementation
+   }
+   
+   public final class Address {
+       private String street;
+       private String city;
+       private String zip;
+   
+       public Address() {
+       }
+   
+       public String getStreet() {
+           return street;
+       }
+   
+       public void setStreet(final String street) {
+           this.street = street;
+       }
+   
+       public String getCity() {
+           return city;
+       }
+   
+       public void setCity(final String city) {
+           this.city = city;
+       }
+   
+       public String getZip() {
+           return zip;
+       }
+   
+       public void setZip(final String zip) {
+           this.zip = zip;
+       }
+       
+       // Rest of implementation
+   }
+
+Creating a Custom CodecRegistry
+-------------------------------
+
+Before you can use a POJO with the driver, you need to configure the
+``CodecRegistry`` to include a codec to handle the translation to and
+from BSON for your POJOs. The simplest way to do that is to use
+the ``PojoCodecProvider.builder()`` method to create and configure a
+``CodecProvider``.
+
+The following example combines the default codec registry, with
+the ``PojoCodecProvider`` configured to automatically create POJO
+``Codec`` instances:
+
+.. code-block:: java
+
+   CodecRegistry pojoCodecRegistry = fromRegistries(
+       MongoClientSettings.getDefaultCodecRegistry(),
+       fromProviders(PojoCodecProvider.builder().automatic(true).build())
+   );
+
+.. note::
+ 
+   The registries are checked in order until one returns a codec for the
+   requested class. The ``DefaultCodecRegistry`` should be first in the
+   list, and the ``PojoCodecProvider`` should always be the last
+   ``CodecProvider`` since it can provide a codec for almost any class.
+
+Using the CodecRegistry
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The following list describes ways to set the ``pojoCodecRegistry`` for
+use:
+
+- Set it when instantiating a ``MongoClient`` object:
+
+.. code-block:: java
+
+   MongoClientSettings settings = MongoClientSettings.builder()
+       .codecRegistry(pojoCodecRegistry)
+       .build();
+   MongoClient mongoClient = MongoClients.create(settings);
```

Comment:
S: since this code example relates to the bullet point, I'd indent it
```suggestion
  .. code-block:: java

     MongoClientSettings settings = MongoClientSettings.builder()
         .codecRegistry(pojoCodecRegistry)
         .build();
     MongoClient mongoClient = MongoClients.create(settings);
```

Reply:
Applies to the next two bullets as well